air-gradient-open-air.yaml

```diff
@@ -0,0 +1,311 @@
+# Airgradient Open Air Outdoor v1.1 presoldered
+# 
+# @llamagecko, @Hendrik and @spectrumjade from AirGradient forum
+# did most of the work for this integration, see the following thread:
+# https://forum.airgradient.com/t/outdoor-monitor-esphome-configuration/823
+#
+
+substitutions:
+  id: "1"
+  devicename: "airgradient-open-air"
+  upper_devicename: "AirGradient Open Air"
+
+esphome:
+  name: "${devicename}-${id}"
+  friendly_name: "${upper_devicename} ${id}"
+  # Automatically add the mac address to the name
+  # so you can use a single firmware for all devices
+  # name_add_mac_suffix: true
+  platform: esp32
+  board: esp32-c3-devkitm-1
+
+# Enable logging
+logger:
+  baud_rate: 0
+
+# Enable Home Assistant API
+api:
+
```

Comment:
It might be nice to include some commented-out info for the encryption key here, e.g.:

```
api:
#  encryption:
#    key: !secret home_assistant_encryption_key
```